src/analysis/network/qgsgraphanalyzer.h

```diff
@@ -38,7 +38,38 @@ class ANALYSIS_EXPORT QgsGraphAnalyzer
      * \param resultTree array that represents shortest path tree. resultTree[ vertexIndex ] == inboundingArcIndex if vertex reachable, otherwise resultTree[ vertexIndex ] == -1
      * \param resultCost array of the paths costs
      */
-    static void dijkstra( const QgsGraph *source, int startVertexIdx, int criterionNum, QVector<int> *resultTree = nullptr, QVector<double> *resultCost = nullptr );
+    static SIP_PYLIST void dijkstra( const QgsGraph *source, int startVertexIdx, int criterionNum, QVector<int> *resultTree = nullptr, QVector<double> *resultCost = nullptr );
```

Comment:
you can rather used `static void SIP_PYTYPE(SIP_PYLIST)` 
I just reanmed SIP_PYARGTYPE to SIP_PYTYPE, so you'll need to rebase